examples/docs/src/pages/themes/docs/configuration.mdx

```diff
@@ -238,6 +238,25 @@ export default {
 }
 ```
 
+## `defaultMenuCollapseLevel`
+
+Specifies the folder level at which the menu on the left is collapsed by default.
+
+**Type:** `number`
+
+**Default:** `1`
+
+**Example:**
+
+```jsx filename="theme.config.js"
+/**
+ * @type {import('nextra-theme-docs').DocsThemeConfig}
+ */
+export default {
+  defaultMenuCollapseLevel: 2
+}
+```
+
```

Comment:
Thanks for the docs! Can you move it to `./docs` folder, because `example/docs` it's not a real Nextra v2 docs but just an example